### PR TITLE
Ensuring 2.0 compatibility

### PR DESCRIPTION
**Pull request by Witus13**
Made necessary changes to make element work in Polymer 2.0 and 1.x

### CODE REVIEW

File: paper-expansion-panel.html [hunk 1]
```diff
@@ -37,24 +37,24 @@
...
 			}
 
 			.toggle {
 				color: var(--disabled-text-color);
 			}
 
 			.content {
-				@apply(--paper-font-body1);
-				@apply(--paper-expansion-panel-content);
+				@apply --paper-font-body1);
```
Comment (ankon) [inline paper-expansion-panel.html]: There is now a closing parentheses left over here.

**Comment by ankon** [comment]
How does this affect compatibility? Will existing users be able to use the element with their existing code bases unchanged, or is this a breaking change?

**Comment by Witus13** [comment]
Oh, sorry, I missed this one.

If I understand the upgrade guide correctly, Polymer 1.7 should be compatible with all changes I made. 1.7 didn't brought any breaking changes as far as I know. I will try this locally tomorrow as it's night here.

**Comment by Witus13** [comment]
I tested it on Chrome 55 (it's the only browser I have, I'm on Chromebook) and it works in Polymer 1.7 and 2.0. I noticed there's a bug with styling in 2.0 but I think it's caused by demo helpers.
If you don't want to cause issues for existing users you can put it on 2.0-preview branch.

File: paper-expansion-panel.html (hunk 1 above)
Comment (Witus13) [inline paper-expansion-panel.html]: Fixed

**Comment by ronnyroeller** [comment]
Thanks for your contribution @Witus13

**Pull request merged**